package/network/config/ltq-vdsl-app/src/src/dsl_cpe_ubus.c

```diff
@@ -146,21 +146,23 @@ static inline void m_bool(const char *id, bool value) {
 }
 
 static inline void m_u32(const char *id, uint32_t value) {
-	blobmsg_add_u32(&b, id, value);
+	blobmsg_add_u64(&b, id, value);
```

Comment:
This isn't resolved, still needs a rebase

Reply:
On the rebasing I could use a little help. Last time I tried this it ended up in a kind of mess.

Reply:
You can continue with the cleanup and fixes I did on my branch.
Assuming you're on the branch from this PR, commit and then backup your changes in a new branch:
`git branch bak-dsl-ubus-stats`

And then switch over:
```
git remote add dhewg https://github.com/dhewg/openwrt.git
git fetch dhewg
git reset --hard dhewg/dsl-ubus-stat
```